tests/vmi_multus_test.go

```diff
@@ -874,6 +866,45 @@ var _ = Describe("SRIOV", func() {
 		It("[test_id:3957]should connect to another machine with sriov interface over IPv6", func() {
 			pingThroughSriov("fc00::1/64", "fc00::2/64")
 		})
+
+		Context("VLANed SR-IOV network", func() {
+			var vlanedVMI1 *v1.VirtualMachineInstance
+			cidrVlaned1 := "192.168.0.1/24"
+			cidrVlaned2 := "192.168.0.2/24"
+			cidrFlat := "192.168.0.3/24"
+			BeforeEach(func() {
+				createNetwork("sriov-vlan", tests.NamespaceTestDefault, sriovConfVlanCRD)
```

Comment:
I'm not sure it is possible. Can you create the networks `BeforeAll` the tests in this `Context`?